### PR TITLE
Adding initialization comments to the store logs

### PR DESCRIPTION
**Pull request by thushara35**
According to the conclusions of code review - "Code review and demo on APIM 2.0 Application and Subscription pages"

### CODE REVIEW

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/extensions/store/app/am-store-apis/apis/application.jag [hunk 1]
```diff
@@ -18,7 +18,7 @@
  *
  */
 require('/modules/store.js').exec(function (ctx) {
-    var log = new Log();
+    var log = new Log('application_endpoint');
```
Comment (cnapagoda) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/extensions/store/app/am-store-apis/apis/application.jag]: Is this log file name('application_endpoint') correct?

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/extensions/store/app/am-store-apis/apis/api-subscriptions.jag [hunk 2]
```diff
@@ -18,7 +18,7 @@
  *
  */
 require('/modules/store.js').exec(function (ctx) {
-    var log = new Log();
+    var log = new Log('api_subsciptions_endpoint');
```
Comment (cnapagoda) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/extensions/store/app/am-store-apis/apis/api-subscriptions.jag]: Is this log file name('api_subsciptions_endpoint') correct?

Comment (thushara35) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/extensions/store/app/am-store-apis/apis/api-subscriptions.jag]: Hi Chandana,

This was discussed in one of my code reviews and this was renamed as per Udara's comment.

Thanks,

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/extensions/store/app/am-store-apis/apis/application.jag (hunk 1 above)
Comment (thushara35) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/extensions/store/app/am-store-apis/apis/application.jag]: Hi Chandana,

This is was discussed in one of my code reviews and this was renamed as per Udara's comment.

Thanks,

**Pull request merged**